Automattic/woocommerce-payments
# Fix Express Checkout Button Design Issues

Fixes https://github.com/Automattic/woopay/issues/1854.

#### Changes proposed in this Pull Request

This changes the design around the express checkout buttons and relevant buttons at the product and cart pages:

-  Buttons are vertically aligned.
- "Add to Cart" and "Proceed to Checkout" buttons get precedence and are listed on top of other buttons.
- The OR separator is removed from the product and cart pages (remains on checkout page).








#### Testing instructions





Check out the express checkout buttons +
-  "Add to cart" button on the product page.
- "Proceed to checkout" button in the cart.
- "Place order" button at checkout.

Confirm all of the aforementioned buttons look OK and similar in those pages. Our base reference is the Storefront theme, but tests on other themes are appreciated. The requirements are:

-  Buttons are vertically aligned.
- "Add to Cart" and "Proceed to Checkout" buttons get precedence and are listed on top of other buttons.
- The OR separator is removed from the product and cart pages (remains on checkout page).

Note: if you find buttons placed weirdly in other themes and that's caused by the theme's CSS, that's alright, as we're not looking to override the theme's styles.

### Screenshots

#### Storefront
- Product page
<img width="797" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/cc9a1df2-fb1a-4320-ba77-e15c97f5f9a1">

- Cart
<img width="802" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/883cc89e-2c0c-4b0a-b66c-a7b53b0a6b8d">

- Checkout
<img width="801" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/d7e3d29b-eeff-4ef5-873e-53fde3e30555">


#### Astra
- Product page (we have a theme conflict here – element with `float: left`)
<img width="1209" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/ebc0ebf7-86d6-43aa-a0b0-49af36963066">

- Cart
<img width="1216" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/05a398a7-85d1-4984-b015-4400fed5dd88">

- Checkout
<img width="1215" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/101f2f7a-b086-4346-9755-08fa00304b5c">


#### Botiga
- Product page
<img width="1122" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/4f5f9daa-6c17-48a7-ab02-dcc936860a27">

- Cart
<img width="1120" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/29856543-dc47-487a-bb9c-bab7d54a6693">

- Checkout
<img width="1124" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/b620df82-ac4e-4f0b-b6da-0a590dde998d">


#### Minimalist eCommerce
- Product page (we have a WooCommerce conflict here – element with `float: left`)
<img width="1189" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/c77a7aff-674c-43e4-941c-9f5760f2f1fb">

- Cart
<img width="1133" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/3663eb36-e6ad-49af-bb82-70d9534d227e">

- Checkout
<img width="1129" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/c2a93875-503e-4adb-86b3-c87470de731f">


#### Neve
- Product page
<img width="1151" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/9904e8fd-c56f-4cfa-858d-7bfe10a34eb7">

- Cart (we have a theme conflict here – element with `display: flex`)
<img width="1150" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/bd2857ff-02e2-4684-921c-d08680b63b79">

- Checkout
<img width="1160" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/dc39f258-59f7-477d-91dc-f3955e96efb4">


-------------------


**Post merge**
